### ISSUE TITLE
508-defect-1 ❗ Launchblocker [SEMANTIC MARKUP, COGNITION]: Grouped radio buttons MUST share a name 

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#17765]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Josh 

## User Story or Problem Statement

As a keyboard user, I want to be able to navigate radio options using the up and down arrow keys. As a screen reader user, I want to be able to hear how many selections exist in a group.

## Details

The yes and no radio buttons for the question "Is this claim going through the legacy appeals process?" do not share a `name` property. On Firefox, a user is unable to navigate using the up and down arrow keys. 

## Acceptance Criteria

- [ ] The radio buttons share a name

## Environment

* Operating System: Windows
* Browser: Firefox
* Screenreading device: NVDA
* Server destination: staging

## Steps to Recreate

1. Enter `https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/introduction` in browser
2. Start screenreading device listed in Environment
3. Select "disability claim", then tab to the question "Is this claim..."
4. Confirm you are unable to use the up and down arrow keys to navigate through radio options

## Solution (if known)

Add a shared `name` property to the radio buttons.

## WCAG or Vendor Guidance (optional)

* [MDN: input type="radio"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio)

## Screenshots or Trace Logs

<img width="1792" alt="Screen Shot 2020-12-18 at 6 10 28 PM" src="https://user-images.githubusercontent.com/14154792/102675219-96f46200-4166-11eb-8bb9-9e0d382c9583.png">

**Comment by joshkimux** [comment]
@Mottie thanks for getting this fixed so quickly! This one is good to go!
<img width="473" alt="Screen Shot 2020-12-19 at 4 30 05 PM" src="https://user-images.githubusercontent.com/14154792/102700011-759f7e80-4217-11eb-8a89-9dad274e9aca.png">
